frontend/resources/translations/sv.json

```diff
@@ -0,0 +1,72 @@
+{
+  "appName": "AppFlowy",
+  "defaultUsername": "Mig",
+  "welcomeText": "Välkommen till @:appName",
+  "welcomeTo": "Välkommen till",
+  "githubStarText": "Stjärna på GitHub",
+  "subscribeNewsletterText": "Prenumerera på nyhetsbrev",
+  "letsGoButtonText": "Snabbstart",
+  "title": "Titel",
+  "youCanAlso": "Du kan också",
+  "and": "och",
+  "failedToOpenUrl": "Det gick inte att öppna webbadressen: {}",
+  "blockActions": {
+    "addBelowTooltip": "Klicka för att lägga till nedan",
+    "addAboveCmd": "Alt+klicka",
```

Comment:
Same thing here, there is already an sv-SE.json. Please specify the country code for this if there are differences.

Btw, the country code allows us to provide different ways of expressing the same thing using the same language, but in a way that is most native and familiar to a specific country. For example, en-US is english that contains phrases and expressions used commonly in the US, while en-UK is for the United Kingdom.